cmd/controller-manager/app/options/options.go

```diff
@@ -246,6 +250,7 @@ func (o *Options) AddFlags(flags *pflag.FlagSet, allControllers, disabledByDefau
 	flags.BoolVar(&o.EnableClusterResourceModeling, "enable-cluster-resource-modeling", true, "Enable means controller would build resource modeling for each cluster by syncing Nodes and Pods resources.\n"+
 		"The resource modeling might be used by the scheduler to make scheduling decisions in scenario of dynamic replica assignment based on cluster free resources.\n"+
 		"Disable if it does not fit your cases for better performance.")
+	flags.BoolVar(&o.SkipStatusCollection, "skip-status-collection", false, "Skip status collection. By default, it is false which means that resource status will be reflected in federated control-plane.")
```

Comment:
This flag just suspends status aggregation, that is `AggregateStatus` from the flow of the interpreter, but the status is still collected to `Work` and `ResourceBinding`, do you think that is expected? Or would this flag be misleading to people?

Reply:
Makes sense. `--skip-status-aggregation`?

Reply:
Obviously, `--skip-status-aggregation` is much better.

The component flag is a kind of  `global` configuration, once this `--skip-status-aggregation` is set to `true`, that means skip status aggregation to all types of resources. 

So, I'm thinking if we need to provide finer control, for example, introduce some fields to the [PropagationPolicy Suspension](https://github.com/karmada-io/karmada/blob/07e297681c40e9dddce8455a2103684a261ffc64/pkg/apis/policy/v1alpha1/propagation_types.go#L218-L232).

I want to hear your thoughts.

Reply:
In our unique setup, a global flag makes sense because we never want to aggregate status back to the "source"/blue cluster.

But I could see the use-case of it being at the policy level for karmada regular (as a federated-api) install mode.

```
type Suspension struct {
	// Dispatching controls whether dispatching should be suspended.
	// nil means not suspend, no default value, only accepts 'true'.
	// Note: true means stop propagating to all clusters. Can not co-exist
	// with DispatchingOnClusters which is used to suspend particular clusters.
	// +optional
	Dispatching *bool `json:"dispatching,omitempty"`
	
+	// Skip Status Aggregation controls whether collecting status from
+     // member clusters should be skipped.
+	// nil means to not skip, no default value, only accepts 'true'.
+	// Note: true means stop collecting status from all member clusters.
+	// +optional
+	SkipStatusAggregation *bool `json:"skipStatusAggregation,omitempty"`
```

I don't know if there is a use-case to say `skipStatusCollectionFromClusters`

Reply:
That's the idea. 
Let's say, here are two approaches:
- The first one is the global flags `--skip-status-aggregation` 
- The second one is the API way.

> In our unique setup, a global flag makes sense because we never want to aggregate status back to the "source"/blue cluster.

If we go with the second approach, you have to configure it to your each PropagationPolicy, is it a burden for you?

> I don't know if there is a use-case to say skipStatusCollectionFromClusters

I don't think so, so far.

Reply:
OK, then, I would prefer to go with the second approach, as it is more flexible. 
Do you think we can have a proposal to address the possible use case, the effect on performance, the features that rely on the status collection, and so on?